script.js

```diff
@@ -248,4 +264,141 @@ jQuery(document)
       }
     }
   })
+
+let citySearchComboBilling = jQuery('#billing_town_city_search');
```

Comment:
We can't use ES6 features in code that gets served directly to the browser (ie no web pack/rollup and babel transpilation)
Please (for now) only use `var` not `let`/`const`